oe_paragraphs.post_update.php

```diff
@@ -200,3 +201,12 @@ function oe_paragraphs_post_update_10006(array &$sandbox): void {
   $field->set('settings', $settings);
   $field->save();
 }
+
+/**
+ * Change the description of oe_paragraphs_description_list paragraph.
+ */
+function oe_paragraphs_post_update_10007(array &$sandbox): void {
```

Comment:
```
francesco in ~/Projects/web/oe_paragraphs <git:OEL-775>
[19:00:29] ❯❯❯ docker-compose exec web ./vendor/bin/drush updb -y                                                                                                                                                                         ✱
 --------------- ----------- ------------- --------------------------------
  Module          Update ID   Type          Description
 --------------- ----------- ------------- --------------------------------
  oe_paragraphs   10007       post-update   Change the description of
                                            oe_paragraphs_description_list
                                            paragraph.
 --------------- ----------- ------------- --------------------------------


 // Do you wish to run the specified pending updates?: yes.

>  [notice] Update started: oe_paragraphs_post_update_10007
>  [error]  Error: Call to a member function set() on null in oe_paragraphs_post_update_10007() (line 210 of /var/www/html/oe_paragraphs.post_update.php)
```
If I run the update on a website that didn't install oe_paragraphs_description_list paragraph.

Reply:
Same error I commited for the OEL-759. Applying same solution.